spec/spec_helper.rb

```diff
@@ -2,16 +2,13 @@
 
 require 'bundler/setup'
 
-require 'coveralls'
+require 'simplecov'
 require 'pry'
 
 require 'faker-bot'
 
-Coveralls.wear!
-
 SimpleCov.formatters = [
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
+  SimpleCov::Formatter::HTMLFormatter
```

Comment:
Question: 🤔 

What are the implications of removing the `coveralls` gem? My understanding is that SimpleCov will _still_ generate the test coverage; which is sent to Travis CI in the build process.

Reply:
I was actually trying to figure that out myself. I went down a rabbit hole and found that coveralls isn’t actually being maintained, but @vbrazo was added as a collaborator to a more updated fork. Maybe he has some insights into what coveralls actually gives us. 

That said, I looked at how faker handles this and they only rely on simplecov. They don’t use coveralls at all. 

Since I’m going to remove rubocop from Travis, I should be able to keep coveralls. Still curious if it’s necessary, though.

Reply:
Makes sense, yeah. AFAIK, SimpleCov will generate the code coverage and send to code climate which looks after these stats for us. Keen to get Vitor's feedback as well. If it's not in use, I'm happy for us to remove it.